src/api/memberApi.ts

```diff
@@ -1,13 +1,32 @@
 import { useQuery, useMutation, useQueryClient } from 'react-query';
 import axios from 'axios';
-import { ProfileInfo } from './dto';
+import { formatGeneration } from '@utils/converter';
+import { FollowInfo, ProfileInfo } from './dto';
 
 const profileKeys = {
   profileInfo: (memberId: number) => ['profile', 'profileInfo', memberId] as const,
 };
 
 const useGetProfileQuery = (memberId: number) => {
-  const fetcher = () => axios.get(`/members/${memberId}/profile`).then(({ data }) => data);
+  const fetcher = () =>
+    axios.get(`/members/${memberId}/profile`).then(({ data }) => {
+      return {
+        ...data,
+        generation: formatGeneration(data.generation),
+        follower: data.follower.map((followerInfo: FollowInfo) => {
+          return {
+            ...followerInfo,
+            generation: formatGeneration(followerInfo.generation),
+          };
+        }),
+        followee: data.followee.map((followeeInfo: FollowInfo) => {
+          return {
+            ...followeeInfo,
+            generation: formatGeneration(followeeInfo.generation),
+          };
+        }),
+      };
+    });
```

Comment:
```suggestion
    axios.get(`/members/${memberId}/profile`).then(({ data }: { data: ProfileInfo }) => {
      return {
        ...data,
        generation: formatGeneration(data.generation),
        follower: data.follower.map((followerInfo) => {
          return {
            ...followerInfo,
            generation: formatGeneration(followerInfo.generation),
          };
        }),
        followee: data.followee.map((followeeInfo) => {
          return {
            ...followeeInfo,
            generation: formatGeneration(followeeInfo.generation),
          };
        }),
      };
    });
```

하위요소에 직접적인 타입 대신 반환 값에 대한 타입 적용으로도 가능할 것 같네용!

Reply:
오홍! 맞네용!! 감사합니다!